restapi/src/main/java/org/cristalise/restapi/RootAccess.java

```diff
@@ -59,6 +58,20 @@ public Response listResourceRoots(
         return toJSON(resourceRoots, null).build();
     }
 
+    @GET
+    @Path("files/{path}")

```

Comment:
what is the purpose of this new endpoint?

Reply:
it allows the user to download the file directly

Reply:
so that we can give a link that downloads the file

Reply:
@kovax I'll remove this endpoint